ibis/tests/expr/test_window_functions.py

```diff
@@ -48,10 +50,24 @@ def test_value_over_api(alltypes):
     w1 = ibis.window(rows=(0, 1), group_by=t.g, order_by=[t.f, t.h])
     w2 = ibis.window(range=(-1, 1), group_by=[t.g, t.a], order_by=[t.f])
 
-    expr = t.f.cumsum().over(rows=(0, 1), group_by=t.g, order_by=[t.f, t.h])
```

Comment:
I think we have been misinterpreting this scenario because we used the `cumulative_window()` frame constructed in the `cumsum()` call, but we also provided explicit rows window boundaries. I changed the behaviour to raise due to window frame boundary conflict.

Reply:
Yeah, that makes sense to me.